addons/libs/packets/fields.lua

```diff
@@ -2132,12 +2132,13 @@ fields.incoming[0x03C] = L{
     {ref=types.shop_item,       label='Item',               count='*'},         -- 08 -   *
 }
 
--- Price response
--- Sent after an outgoing price request for an NPC vendor (0x085)
+-- Price/sale response
+-- Sent in response to an outgoing price request for an NPC vendor (0x085), and in response to player finalizing a sale.
+-- Inventory bag is assumed to be regular Inventory (0)
 fields.incoming[0x03D] = L{
     {ctype='unsigned int',      label='Price',              fn=gil},            -- 04
     {ctype='unsigned char',     label='Inventory Index',    fn=invp+{0x09}},    -- 08
-    {ctype='unsigned char',     label='Bag',                fn=bag},            -- 09
+    {ctype='unsigned char',     label='Type'},                                  -- 09 0 = on apprasial, 1 = when sale is finalized
     {ctype='unsigned short',    label='_junk1'},                                -- 0A
     {ctype='unsigned int',      label='_unknown1',          const=1},           -- 0C
```

Comment:
`_unknown1` appears to be `Quantity Sold`, can you confirm that and add it to your PR.

Reply:
It'll be a 1 for a Price Check though.

Reply:
Confirming that I did check (with crystals of varying quantities) and `_unknown1` is quantity sold or 1 depending on type of response